config/packages/netgen_notifications.yaml

```diff
@@ -0,0 +1,23 @@
+netgen_notifications:
+    categories:
+        default:
+            subscription:
+                enabled: true
+                resolver: netgen_notifications.ibexa.subtree_update_resolver
+                digest_actions:
+                    email:
+                        type: digest_email
+                        parameters:
+                            from: 'info@netgen.io'
```

Comment:
This should be a parameter (env variable?)

Reply:
Done - https://github.com/netgen/media-site/pull/189/commits/3a4cdb49b1d4770ba552b43bbdb3f79da497ed4f